source/common/runtime/runtime_features.cc

```diff
@@ -111,6 +111,9 @@ constexpr const char* runtime_features[] = {
 constexpr const char* disabled_runtime_features[] = {
     // v2 is fatal-by-default.
     "envoy.test_only.broken_in_production.enable_deprecated_v2_api",
+    // Defaulting to off due to high risk.
+    // TODO(ggreenway): Move this to default-on during 1.20 release cycle.
```

Comment:
I'd be inclined to either get someone to sign on for testing this (apple / setec-canary companies / @wez470) , or just flip after a week or two once we've established the "off" path is still solid.

Reply:
Yeah, I was going to canary internally, and @rgs1 has already run it, so I was going to flip after a week or so.